docs/mobile/events.md

```diff
@@ -15,19 +16,23 @@ All mobile events MUST set `event.domain` as `device`.
 
 ## Lifecycle instrumentation
 
-This section defines how to apply semantic conventions when instrumenting application lifecycle.
-This event is meant to be used in conjunction with `os.name` [resource semantic convention](/docs/resource/os.md) to identify the mobile operating system (e.g. Android, iOS).
+This section defines how to apply semantic conventions when instrumenting
+application lifecycle. This event is meant to be used in conjunction with
+`os.name` [resource semantic convention](/docs/resource/os.md) to identify the
+mobile operating system (e.g. Android, iOS).
 
 ### iOS
 
 <!-- semconv ios.lifecycle.events -->
-The event name MUST be `app.lifecycle`.
+The `event.name` MUST be `device.app.lifecycle`.
```

Comment:
The section between `<!-- semconv ios.lifecycle.events -->` and `<!-- endsemconv -->` is auto-generated.

Please do the actual changes in the corresponding yaml file, i.e. following places:

- https://github.com/open-telemetry/semantic-conventions/blob/126059d99dc29fd9a59893091f56500d4b9674b7/model/logs/mobile-events.yaml#L5
- https://github.com/open-telemetry/semantic-conventions/blob/126059d99dc29fd9a59893091f56500d4b9674b7/model/logs/mobile-events.yaml#L44

And then, please run `make table-generation` to re-generate that table. This will also fix the current error:

> File /spec/general/events.md contains a table that would be reformatted.
make: *** [Makefile:98: table-check] Error 1

Reply:
Ahhh ok thanks @AlexanderWert .